lib/types/src/basetunables.rs

```diff
@@ -0,0 +1,58 @@
+use crate::units::Pages;
+use crate::compilation::target::{PointerWidth, Target};
+
+/// Tunable parameters for WebAssembly compilation.
+/// This is the reference implementation of the `Tunables` trait,
+/// used by default.
+///
+/// You can use this as a template for creating a custom Tunables
+/// implementation or use composition to wrap your Tunables around
+/// this one. The later approach is demonstrated in the
+/// tunables-limit-memory example.
+#[derive(Clone)]
+pub struct BaseTunables {
+    /// For static heaps, the size in wasm pages of the heap protected by bounds checking.
+    pub static_memory_bound: Pages,
+
+    /// The size in bytes of the offset guard for static heaps.
+    pub static_memory_offset_guard_size: u64,
+
+    /// The size in bytes of the offset guard for dynamic heaps.
+    pub dynamic_memory_offset_guard_size: u64,
+}
```

Comment:
Can we move this to the compiler instead (same place where the engine lives?)

Reply:
Oh, it's a left over of a try that I forgot to remove.